test/acceptance/behave/README.md

```diff
@@ -21,15 +29,11 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 - Both if you are using a virtual environment or not:
   * Change to the test/acceptance/behave folder of the project.
   * Install the requirements for the acceptance tests in the virtual environment
+  * You should add "--upgrade" (if you have a previous installed version of this library)
 ```
-     pip install -r requirements.txt --allow-all-external
+     pip install [--upgrade] -r requirements.txt --allow-all-external
 ```
 
-#### Requirements to fabric (http://www.fabfile.org/)
-```
-     yum install gcc python-devel
-```
-   Fabric is a Python (2.5-2.7) library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
 
 #### Folders/Files Structure
 
```

Comment:
However, something is failing, as otherwise, I wouldn't get that error and that error wouldn't disappear when I manually create the `logs/` directory.